### DISCUSSION TITLE
latest libuv cannot run in macos 10.x

### DISCUSSION THREAD

**Discussion by yushizhan** [libuv/libuv#3739] [Q&A] [answered]
Dyld Error Message:
  Symbol not found: ___darwin_check_fd_set_overflow
  Referenced from: /Users/USER/Desktop/*/MacDemo.app/Contents/MacOS/MacDemo
  Expected in: /usr/lib/libSystem.B.dylib
 in /Users/USER/Desktop/*/MacDemo.app/Contents/MacOS/MacDemo

**Comment by bnoordhuis** [comment, marked as answer]
That's an Xcode issue, not a libuv issue. Google around for the symbol name, plenty of people have hit it before.

**Comment by yushizhan** [comment]
thx。
